assets/locales/fr.po

```diff
@@ -63,7 +63,7 @@ msgid "Login Forgot password"
 msgstr "Mot de passe oublié ?"
 
 msgid "Authorize Title"
-msgstr "Autoriser %s à accéder à votre profil ?"
+msgstr "Autoriser %s à accéder à votre profil ?"
```

Comment:
oh the stack uses Transifex too ? I thought not because the translations files are versioned unlike the apps we do, but I'll do it right away.